src/Exception/DispatchingException.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace Equip\Exception;
+
+use InvalidArgumentException;
+
+class DispatchingException extends InvalidArgumentException
+{
+    /**
+     * @param string $spec
+     *
+     * @return static
+     */
+    public static function notInvokable($spec)
+    {
+        return new static(sprintf(
+            'Dispatcher `%s` is not invokable',
+            $spec
```

Comment:
Ah, point. More ☕ . Though... it appears the same applies to objects of classes that don't define `__toString()`.

```
$ php -r 'class foo { } var_dump((string) new foo);'
PHP Catchable fatal error:  Object of class foo could not be converted to string in Command line code on line 1
```